programs/uxd/tests/integration_tests/api/program_uxd/instructions/process_mint_with_credix_lp_depository.rs

```diff
@@ -150,13 +150,15 @@ pub async fn process_mint_with_credix_lp_depository(
     .map_err(program_test_context::ProgramTestError::Anchor)?;
     let fees_amount = collateral_amount - redeemable_amount;
 
-    // Check result
+    // redeemable_mint.supply must have increased by the minted amount
```

Comment:
super nit: 
could add a line saying minted amount == redeemable_amount, 
or just add as `let minted_amount = redeemable_amount`,
just be as explicit as possible for the purpose of clarification